Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish.Tests/UnitTests.cs

```diff
@@ -258,9 +259,9 @@ public void TestURIConnection()
             }
         };
 
-        var readValues = new ReadValues();
+        var readValues = new Helper.ReadValues();
         var result = RabbitMQ.Publish(input, connection);
-        ReadMessage(readValues, connection);
+        Helper.ReadMessage(readValues, connection);
 
         Assert.IsTrue(!string.IsNullOrEmpty(readValues.Message) && readValues.Message.Equals("test message"));
         Assert.IsTrue(result.DataFormat.Equals("String")
```

Comment:
Please move these checks to their own asserts and use `AreEqual` for equal comparisons.